tests/conftest.py

```diff
@@ -93,6 +93,53 @@ def _patch_stdio_client() -> Generator[
         yield patched, mock_transport
 
 
+def check_multi_tool_usage_google(json_logs: dict):
+    print("google tool usage check")
+
+
+def check_multi_tool_usage_langchain(json_logs: List[Dict]):
+    # find trace that has 
+    for json_log in json_logs:
+        attrs = json_log["attributes"]
+        name = json_log["name"]
+        if "metadata" in attrs:
+            metadata = json.loads(attrs["metadata"])
+            if attrs["openinference.span.kind"] == "AGENT" and metadata["langgraph_step"] == 2 and name.startswith("transfer_to_"):
+                return
+    raise Exception("")
```

Comment:
This type of extraction belongs in the src/any_agent/telemetry code. All access of trace specific info should be interfaced through those classes imo

Reply:
I'd argue that this is test code, no src code, so a separate function should be used.